content/fr/_index.html

```diff
@@ -1,5 +1,5 @@
 ---
-title: "La meilleure solution d'orchestration de conteneurs en production"
+title: "Solution d'orchestration de conteneurs, prête pour production"
 abstract: "Déploiement, mise à l'échelle et gestion automatisés des conteneurs"
```

Comment:
abstract: "Déploiement, mise à l'échelle et gestion automatisée des conteneurs"

Reply:
@mhamdisemah I can do a new pull request for that. This PR is just about the slogan.

Reply:
I've revised